debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -737,7 +737,7 @@ public static GtidNewChannelPosition parse(String value, String defaultValue) {
             .withType(Type.INT)
             .withWidth(Width.SHORT)
             .withImportance(Importance.MEDIUM)
-            .withDescription("Maximum time in milliseconds to wait after trying to connect to the database before timing out.")
+            .withDescription("A delay period after trying to connect to the database before timing out (given in milliseconds). Defaults to 30 seconds (30000 ms)")
```

Comment:
Hum hum; wasn't "Maximum time to wait after..." the better wording? There might be a misunderstanding, I didn't mean to suggest to literally put "A delay period" everywhere, but rather to replace those usages like "The number of milliseconds" with a more semantical description of what it is. "Maximum time to wait after..." is such semantical wording IMO.